<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>filter_results_window</class>
 <widget class="QWidget" name="filter_results_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyBookMgr - Filter Results</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Results of your filter:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTableWidget" name="filter_results_table">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Title</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Author</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Rating</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Genre</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Series</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Notes</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="close_Button">
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CLOSE</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
